test/driver_test.rb

```diff
@@ -1,39 +1,39 @@
 require_relative 'test_helper'
 
-xdescribe "Driver class" do
-  describe "Driver instantiation" do
+describe 'Driver class' do
+  describe 'Driver instantiation' do
     before do
       @driver = RideShare::Driver.new(
         id: 54,
-        name: "Test Driver",
-        vin: "12345678901234567",
+        name: 'Test Driver',
+        vin: '12345678901234567',
         status: :AVAILABLE
       )
```

Comment:
Consider moving this driver outside the `Driver instantiation` describe block so that you can use if for other tests.